nautobot/docs/installation/http-server.md

```diff
@@ -138,16 +138,47 @@ Finally, restart the `nginx` service to use the new configuration.
 $ sudo systemctl restart nginx
 ```
 
+!!! info
+    If the restart fails, and you changed the default key location, check to make sure the `nautobot.conf` file you pasted has the updated key location. For example, CentOS 8.2.2 requires keys to be in `/etc/pki/tls/` instead of `/etc/ssl/`.
+
+## Confirm Permissions for /opt/nautobot
+
+Ensure that the `/opt/nautobot/` permissions are set to `755`.
+
+Example of correct permissions:
+```no-highlight
+[root@localhost ~]# ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+[root@localhost ~]# 
+```
+
+If the permissions are not correct, modify them accordingly.
+
+Example of modifying the permissions:
+```no-highlight
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwx------. 11 nautobot nautobot 4096 Apr  5 10:00 nautobot
+[nautobot@localhost ~]$ chmod 755 /opt/nautobot/
+[nautobot@localhost ~]$ ls -l /opt/
+total 4
+drwxr-xr-x. 11 nautobot nautobot 4096 Apr  5 11:24 nautobot
+```
+
 ## Confirm Connectivity
 
-At this point, you should be able to connect to the HTTPS service at the server name or IP address you provided.
+At this point, you should be able to connect to the HTTPS service at the server name or IP address you provided. The port number is no longer necessary in the URL in the base config.
```

Comment:
What port number is being referred to here?

Reply:
The port number when we were testing the server at 0.0.0.0:8000.  Meh, I was on the fence about this one.  I just removed this addition, unless someone can think of a concise way to state that which does not cause confusion.